rs/sns/testing/README.md

```diff
@@ -1,8 +1,49 @@
 # SNS testing
 
+## Prerequisites
+
+SNS testing CLI uses identities managed by `dfx`. Currently, the scenario requires two identities:
+the first one represents developer identity and is used to submit NNS/SNS proposals, and the second one
+that is used to distribute ICP tokens to the SNS swap participants
+
+We suggest creating dedicated identities to use with this scenario:
+```
+dfx identity new sns-testing --storage-mode=plaintext
+dfx identity new icp-treasury --storage-mode=plaintext
+```
+Mind the `--storage-mode=plaintext` as `dfx-core` used requires reading `.pem` files from the disk.
```

Comment:
We should avoid referring to dfx-core in the docs, as that's an implementation detail.